docs/tutorial/03-resources.mdx

```diff
@@ -0,0 +1,486 @@
+In real applications, we would likely perform necessary checks and actions with the location path we are storing in
+to make sure we don't abort a transaction because of an accidental overwrite.
+
+Now that you have executed the transaction, account `0x01` should have the newly created `HelloWorld.HelloAsset`
+resource stored in its storage.
+
+### Load Hello Transaction
+
+Now we're going to use a transaction to call the `hello()` method from the `HelloAsset` resource.
+
+<Callout type="info">
+
+Open the transaction named `Load Hello`.
+
+<br />
+
+`Load Hello` should contain the following code:
+
+</Callout>
+
+```cadence:title=LoadHello.cdc
+import HelloWorld from 0x01
+
+// This transaction calls the "hello" method on the HelloAsset object
+// that is stored in the account's storage by removing that object
+// from storage, calling the method, and then putting it back in storage
+
+transaction {
+
+    prepare(acct: AuthAccount) {
+
+        // Load the resource from storage, specifying the type to load it as
+        // and the path where it is stored
+        let helloResource <- acct.load<@HelloWorld.HelloAsset>(from: /storage/HelloAssetTutorial)
+
+        // We use optional chaining (?) because the value in storage
+        // may or may not exist, and thus is considered optional.
+        log(helloResource?.hello())
+
+        // Put the resource back in storage at the same spot
+        // We use the force-unwrap operator `!` to get the value
+        // out of the optional. It aborts if the optional is nil
+        acct.save(<-helloResource!, to: /storage/HelloAssetTutorial)
+    }
+}
+```
+
+Here's what this transaction does:
+
+1. Import the `HelloWorld` definitions from account `0x01`
+2. Moves the `HelloAsset` object from storage to `helloResource` with the move operator and the `load` function from the [account storage API](https://docs.onflow.org/cadence/language/accounts/#account-storage-api)
+3. Calls the `hello()` function of the `HelloAsset` resource stored in `helloResource` and logs the result
+4. Saves the resource in the account that we originally moved it from at the path `/storage/HelloAssetTutorial`
+
+We're going to be using the `prepare` phase again to load the resource because it
+has access to the signing accounts' [private `AuthAccount` object](/cadence/language/accounts/#authaccount).
+
+Let's go over the transaction in more detail.
+To remove an object from storage, we use the `load` method from the [account storage API](https://docs.onflow.org/cadence/language/accounts/#account-storage-api)
+
+```cadence
+let helloResource <- acct.load<@HelloWorld.HelloAsset>(from: /storage/HelloAssetTutorial)
+```
+
+If no object of the specified type is stored under the given path, the function returns `nil`. (This is an [Optional](/cadence/language/values-and-types/#optionals),
```

Comment:
With Secure Cadence, if the object at the given path is not of the specified type, Cadence will throw an error. We should probably mention this here.